CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ## 0.20.0
 
 * Updated to Cruise Control 2.0.124, which fixes a previous issue with CPU utilization statistics for containers. As a result, the CPUCapacityGoal has now been enabled.
+* Removed deprecated metrics port 9404 and Prometheus annotations from services
```

Comment:
Can you similarly to 0.19.0 release add special section for deprecations and removals and add it there? It can get easily lost here. You should probably also mention the removed services and the option to ise templates for adding such annotations.